editor/modes/visual-editor/block.js

```diff
@@ -129,7 +128,8 @@ class VisualEditorBlock extends wp.element.Component {
 					}
 				},
 				...blocksWithTheSameType.slice( 1 )
-			]
+			],
+			{ offset: -1 }
```

Comment:
It feels a bit weird to include the `focus` update in the `replaceBlocks` actions. Is this necessary?